CHANGELOG.rst

```diff
@@ -4,6 +4,7 @@ new version (on deck)
 
 v0.18.2
 =======
+- [ENH] Series toset() functionality #570
 - [INF] Make requirements.txt smaller
```

Comment:
@eyaltrabelsi I think both lines 7 and 8 should have been put inside the "new version (on deck)" section, as 0.18.2 has already been released.

Can you move them please?

Reply:
@ericmjl I will move them, also i was creating the docs locally and couldn't find toset although i think i have added  all changes required